### PR TITLE
actionified, formatted and doxy'd

### PR DESCRIPTION
**Pull request by siddacious**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Adafruit DotStar [![Build Status](https://github.com/adafruit/Adafruit_DotStar/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_DotStar/actions)
 # Adafruit DotStar Library [![Build Status](https://travis-ci.com/adafruit/Adafruit_DotStar.svg?branch=master)](https://travis-ci.com/adafruit/Adafruit_DotStar)
```
Comment (ladyada) [inline README.md]: remove this line

**Review by ladyada — APPROVED** [review]

**Comment by PaintYourDragon** [comment]
Hey question. Not especially DotStar-related, but this just popped up in my mailbox and there’s been a thing on my mind…

1. Do we have a guide or any documentation on the entire start-to-finish process of Actionifying, formatting and doxying? [Like we had for Travis CI before](https://learn.adafruit.com/the-well-automated-arduino-library). I’d really like to understand how to set up the whole process from the start so I’m not dependent on others to wave magic wands and then I come back and some mysterious stuff has happened and I’m either confused, annoyed and/or break it.
2. If not the full deal, could you point me toward a specific reference on the source code formatting being used, so I can just follow that from the get-go instead of writing things my own way and then inflicting them on you to reformat? (I mean yeah, I could just look at some of the output, but it’s more thorough to have a full standard to refer to.)

Much thanks.

**Comment by ladyada** [comment]
i havent updated the guide for actions (maybe if @siddacious is feeling industrious he can do so :) 

heres my 'actions' checklist

* git pull
* create new branch actionsci
* switch to branch actionsci
* cp Adafruit_ILI9341/.github -> repo
* in githubci.yml update PRETTYNAME
* in repo add GH_REPO_TOKEN to secrets
* move any travis library installs to library.properties "depends=" list
* delete .travis.yml
* within repo run: clang-format.exe -i *.cpp *.h
* bump library.properties
* change readme.md badge to (update reponame)

```[Build Status](https://github.com/adafruit/Adafruit_ILI9341/actions)```

**Comment by siddacious** [comment]
Ya, I've been meaning to update the guide. I'll see about taking a pass on Monday

**Comment by siddacious** [comment]
@PaintYourDragon in the meanwhile we're still using doxygen for documentation generation as documented in the guide:
https://learn.adafruit.com/the-well-automated-arduino-library/doxygen

For code formatting we're using `clang-format` as Limor mentioned. It'll to the formatting automagically without modifying the behavior

**Comment by PaintYourDragon** [comment]
Much appreciated. Thanks! 🐲

**Pull request merged**